src/main/java/edu/ie3/datamodel/models/value/load/BdewLoadValues.java

```diff
@@ -263,10 +265,11 @@ public static BdewSeason parse(String key) throws ParsingException {
         case "Wi", "Winter" -> WINTER;
         case "Su", "Summer" -> SUMMER;
         case "Tr", "Intermediate" -> TRANSITION;
-        default -> throw new ParsingException(
-            "There is no season for key:"
-                + key
-                + ". Permissible keys: 'Wi', 'Winter', 'Su', 'Summer', 'Tr', 'Intermediate'");
+        default ->
+            throw new ParsingException(
+                "There is no season for key:"
+                    + key
+                    + ". Permissible keys: 'Wi', 'Winter', 'Su', 'Summer', 'Tr', 'Intermediate'");
```

Comment:
Do you know why we used `intermediate` here?

Reply:
I have no idea. Should be Transition, right?

But this is nothing for this PR. This should just contain the reformating.

Reply:
That's right. I just saw this.